tests/e2e/utils/helpers.js

```diff
@@ -0,0 +1,56 @@
+const axios = require('axios')
+const https = require('https')
+const fs = require('fs')
+const path = require('path')
+
+const { config } = require('../cucumber.conf')
+
+const baseUrlOcis = config.baseUrlOcis
+
+const createdFiles = []
+
+const makeApiRequest = async (method, url, headers, data) => {
+  try {
+    await axios({
+      method,
+      url,
+      headers,
+      data,
+      httpsAgent: new https.Agent({ rejectUnauthorized: false })
+    })
+  } catch (error) {
+    console.error('API request failed:', error.message)
+    throw error
+  }
+}
+
+const getAuthHeaders = () => {
+  const headers = {
+    Authorization: `Basic ${Buffer.from(`${config.adminUser}:${config.adminPassword}`).toString(
+      'base64'
+    )}`,
+    'Content-Type': 'text/markdown'
+  }
+  return headers
+}
+
+const uploadFileToServer = async (fileName, method) => {
```

Comment:
why is the `method` parameter needed here? an upload always uses `PUT`

Reply:
since `makeApiRequest()` is dynamic and takes `method` as parameter, the function `uploadFileToServer()` is also made to take `method` as parameter. But passed `PUT` directly as parameter in the `makeApiRequest` inside `uploadFileToServer` now.